src/Ryujinx.Common/Configuration/AppDataManager.cs

```diff
@@ -118,9 +112,125 @@ public static void Initialize(string baseDirPath)
             SetupBasePaths();
         }
 
+        public static string GetOrCreateLogsDir()
+        {
+            if (Directory.Exists(LogsDirPath))
+            {
+                return LogsDirPath;
+            }
+            Logger.Notice.Print(LogClass.Application, "Logging directory not found; attempting to create new logging directory.");
```

Comment:
```suggestion
            }

            Logger.Notice.Print(LogClass.Application, "Logging directory not found; attempting to create new logging directory.");
```

You missed the extra empty line after the `{`. (It's just for consistencies)